public/javascripts/NumberGuesser.js

```diff
@@ -0,0 +1,27 @@
+class NumberGuesser {
+    constructor() {
+    this.clear()
+  }
+  attempts(guess) {
+    if(guess !== 'exit') {
+     this.guesses.push(guess, this.compare(guess))
+   }
+  }
+
+  clear() {
+    this.answer = Math.ceil(Math.random() * 100)
+    this.guesses = []
+  }
+
+  compare(guess) {
+    if(guess === 'exit') {return}
```

Comment:
The formatting and indentation of this function makes it difficult to read.  I recommend using explicit braces for blocks, and never having single line control flow constructs to increase both readability and maintainability.